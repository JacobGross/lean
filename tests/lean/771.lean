set_option pp.all true
definition id_1 (n : nat) :=
   by exact n

definition id_2 (n : nat) :=
   ((by exact n) : nat)

definition id_3 (n : nat) : nat :=
by exact n

definition id_4 (n : nat) :=
show nat, by exact n
