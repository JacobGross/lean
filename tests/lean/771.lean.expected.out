771.lean:3:3: error: failed to synthesize placeholder, type is a unknown (i.e., it is a metavariable) (solution: provide type explicitly)
